src/constants/index.js

```diff
@@ -17,6 +17,9 @@ export const KUBEVIRT_API_VERSION = 'kubevirt.io/v1alpha3';
 export const OS_LABEL = 'kubevirt.io/os';
 export const POD_NETWORK = 'Pod Networking';
 
+export const DEVICE_TYPE_INTERFACE = 'interfaces';
```

Comment:
I know this is just an id but does it have to be plural?

Reply:
please look at the comment in `patches.js`

Reply:
Agreed and done!